.gitignore

```diff
@@ -7,3 +7,4 @@ prototypes
 _build
 testing
 CytoSnake.egg-info
+workflows/rules/common.py
```

Comment:
Seeing this line made me wonder if additional rules might have related python files which should be ignored. If so, consider adding them here.

Reply:
This is not a workflow script. I convert `common.smk` to `common.py` in order to test whether the generated pathing is correct. Since snakemake uses python syntax. converting it to a `common.smk` to `common.py` is python executable. 

Let me know if this does not make sense!